config/configtest/config.yaml

```diff
@@ -0,0 +1 @@
+port: 2424
```

Comment:
Consider naming this file differently, such as `config_with_port.yaml`.

Reply:
I guess it's better to change the name to `config/configtest/config_same_as_defaults.yaml`